File: app/src/main/java/com/o4x/musical/ui/dialogs/DiscardTagsDialog.java
package com.o4x.musical.ui.dialogs;

import android.app.Dialog;
import android.app.DialogFragment;
import android.os.Bundle;

import androidx.annotation.NonNull;

import com.afollestad.materialdialogs.MaterialDialog;
import com.o4x.musical.R;

public class DiscardTagsDialog extends DialogFragment {


    public static DiscardTagsDialog create() {
        return new DiscardTagsDialog();
    }

    @NonNull
    @Override
    public Dialog onCreateDialog(Bundle savedInstanceState) {
        MaterialDialog.Builder builder =
                new MaterialDialog.Builder(getActivity())
                        .content(R.string.discard_changes)
                        .neutralText(android.R.string.cancel)
                        .positiveText(android.R.string.ok)
                        .onNeutral((dialog, which) -> {})
                        .onPositive((dialog, which) -> getActivity().finish());
        return builder.build();
    }

}

File: app/src/main/java/github/o4x/musical/ui/dialogs/DiscardTagsDialog.kt
package github.o4x.musical.ui.dialogs


import android.app.Dialog
import android.os.Bundle
import androidx.fragment.app.DialogFragment
import com.afollestad.materialdialogs.MaterialDialog
import github.o4x.musical.R

class DiscardTagsDialog : DialogFragment() {
    override fun onCreateDialog(savedInstanceState: Bundle?): Dialog {
        return MaterialDialog(requireContext())
            .message(R.string.discard_changes)
            .neutralButton(R.string.cancel)
            .positiveButton(R.string.ok) {
                activity?.finish()
            }
    }

    companion object {
        fun create(): DiscardTagsDialog {
            return DiscardTagsDialog()
        }
    }
}